src/main/java/jenkinsci/plugins/modernstatus/PluginImpl.java

```diff
@@ -39,6 +39,11 @@ public void start() throws Exception {
         wrapperField.setAccessible(true);
         wrapper = (PluginWrapper) wrapperField.get(this);
     } catch (Exception e) {}
+    try {
+      Field colorValue = Color.class.getDeclaredField("value");
```

Comment:
I would rather not do this, it adds warnings on java 9+.

I'm going to close this now given it's really old, if others really want it then please re-submit